src/main/java/com/conveyal/taui/controllers/SinglePointAnalysisController.java

```diff
@@ -42,6 +42,8 @@
         Project project = Persistence.projects.findByIdIfPermitted(analysisRequest.projectId, accessGroup);
         TravelTimeSurfaceTask task = (TravelTimeSurfaceTask) analysisRequest.populateTask(new TravelTimeSurfaceTask(), project);
 
+        if (req.headers("Accept").equals("image/tiff")) task.format = TravelTimeSurfaceTask.Format.GEOTIFF;
```

Comment:
This seems to be relying on a default initializer to select `GRID`, it might be better to add an `else` clause to make it obvious what happens when the header is not `image/tiff`.